githubatom

```diff
@@ -3,3 +3,4 @@ for macOS, Linux, and Microsoft Windows with support
 for plug-ins written in Node.js, and embedded Git Control,
 developed by GitHub.
 Atom is a desktop application built using web technologies.
```

Comment:
Can we also add the details how to include packages in atom ?

Reply:
Included the requested details

Reply:
looks good..